js/hole.js

```diff
@@ -279,7 +279,9 @@ async function submit() {
     }
 
     const data = await res.json();
-    loggedIn = data.LoggedIn;
+    // For experimental holes, users are always effectively logged-out for autosave purposes,
+    // because solutions are not stored on the server.
+    loggedIn = data.LoggedIn && !data.Experimental;
```

Comment:
Experimental is a property of the hole, not the solution, I'd rather pass it from the template to the JS than return it in the solution response.

Reply:
Good point